app/controllers/plans_controller.rb

```diff
@@ -43,4 +43,21 @@ def search
       # TODO
     end
   end
+
+  def set_autofinger_subscription
+    owner = current_account
+    interest = Account.find_by_username(params[:id])
+    autofinger = owner.interests_in_others.find_or_create_by_interest(interest.id)
+    success = autofinger.update_attributes(priority: params[:priority])
+    if success
+      if params[:priority] == '0'
+        notice = 'User was removed from your autoread list.'
+      else
+        notice = "User is now on your autoread list with priority level of #{params[:priority]}."
+      end
+    else
+      notice = "Could not change autoread priority. Please try again."
```

Comment:
I think just changing "please try again" to something like "if this happens more than once, seek help" would be enough. It shouldn't ever happen, and if it does the best thing to have happen would be to have someone report it.